Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -20,16 +20,31 @@ namespace Microsoft.Toolkit.Services.MicrosoftGraph
     /// <summary>
     /// Authentication Helper Using Azure Active Directory v2.0 app Model
     /// </summary>
-    internal class MicrosoftGraphAuthenticationHelper
+    public class MicrosoftGraphAuthenticationHelper
```

Comment:
Not sure if this type should be made public? It was fine to link it into the other project where it was used.

Reply:
Agreed. In Win32 projects we have quite a bit of this, and have a new folder (named Shared) where all this code belongs. We then just bring in all the shared items with one include in the csproj, or cherry pick individual files.

Reply:
Just changed it back, didn't realize you had it linked already